### PR TITLE
#680 Cypress e2e test broker with service page

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #680 )
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/integration/brokerPage.test.js [hunk 1]
```diff
@@ -14,26 +14,32 @@
  * limitations under the License.
  */
 
-import { KAFKA } from '../../src/constants/urls';
+import { BROKER } from '../../src/constants/urls';
 
 // TODO: move the tests to configuration modal
```
Comment (konekoya) [inline ohara-manager/client/cypress/integration/brokerPage.test.js]: Please delete the comment as it's no longer relevant :)

File: ohara-manager/client/src/constants/urls.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ export const HOME = '/';
 export const PIPELINE = '/pipelines';
 
 export const KAFKA = '/kafka';
```
Comment (konekoya) [inline ohara-manager/client/src/constants/urls.js]: I think it should be safe to remove `KAFKA` from our codebase now?

File: ohara-manager/client/cypress/integration/brokerPage.test.js [hunk 3]
```diff
@@ -14,26 +14,32 @@
...
+      .click()
+      .type('1');
     cy.getByText('Save').click();
-
     cy.get('.toast-success').should('have.length', 1);
     cy.get('.ReactModal__Content').should('have.length', 0);
     cy.get('td')
       .contains('test cluster')
       .should('have.length', 1);
+    cy.deleteAllTopics();
```
Comment (konekoya) [inline ohara-manager/client/cypress/integration/brokerPage.test.js]: We should do the clean up before running the test suite :)

File: ohara-manager/client/cypress/integration/brokerPage.test.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/cypress/integration/brokerPage.test.js]: ok 
I will remove it

File: ohara-manager/client/src/constants/urls.js (hunk 2 above)
Comment (wu87988622) [reply ohara-manager/client/src/constants/urls.js]: yah 
Is my fault

File: ohara-manager/client/cypress/integration/brokerPage.test.js (hunk 3 above)
Comment (wu87988622) [reply ohara-manager/client/cypress/integration/brokerPage.test.js]: ok I change to page fist function

**Comment by konekoya** [comment]
Thanks for the PR @wu87988622. Please merge this PR if build passes 👍

**Pull request merged**